cluster/src/main/scala/org/apache/spark/memory/SnappyUnifiedMemoryManager.scala

```diff
@@ -580,6 +619,8 @@ object SnappyUnifiedMemoryManager extends Logging {
       math.max(getMaxHeapMemory / 20, 500L * 1024L * 1024L))
   }
 
+  private val DEFAULT_EVICTION_PERCENT = 80f
```

Comment:
Off-heap eviction at 90% seems fair, but IMO heap eviction at 90% causes a lot of activity during query execution, eviction and all creates a lot of garbage and it quickly reaches critical limit. I would suggest lets keep eviction at 80 and once we get more confidence we can change the limit.